src/datastore_mad/remotes/downloader.sh

```diff
@@ -431,11 +431,17 @@ lxd://*)
     file_type="application/octet-stream"
     command="$VAR_LOCATION/remotes/datastore/lxd_downloader.sh \"$FROM\""
     ;;
-restic://*)
-    eval `$VAR_LOCATION/remotes/datastore/restic_downloader.rb "$FROM" | grep -e '^command=' -e '^clean_command='`
+restic://*|restic+rbd://*)
+    defs=`$VAR_LOCATION/remotes/datastore/restic_downloader.rb "$FROM" | grep -e '^command=' -e '^clean_command='`
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The use of backticks for command substitution in shell scripts can lead to vulnerabilities, quoting issues and nested command complexities. To mitigate this issue, it is recommended to use the more secure &#39;$(...)&#39; syntax for command substitution. This method properly quotes the substituted command, which is more readable, easier to nest, and avoids undefined behaviors related to quoting. Adopting this practice improves code clarity and maintainability. For more information on shell scripting best practices, refer to the GNU Bash Manual (https://www.gnu.org/software/bash/manual/bash.html#Command-Substitution).